server/config/index.js

```diff
@@ -32,6 +34,19 @@ function anyEnabled() {
 	} );
 }
 
+const ssrConfig = template(
+	'var data = <%= data %>;' +
+	'<%= config %>' +
+	'<%= isEnabled %>' +
+	'<%= anyEnabled %>'
+) ( {
+	data: JSON.stringify( clientData ),
+	config: config.toString(),
```

Comment:
I branched off a PR to test this out: https://github.com/Automattic/wp-calypso/pull/11454